static/docs/use-cases/shared-storage-on-nfs.md

```diff
@@ -0,0 +1,148 @@
+# Shared Storage on NFS
+
+In the modern software development environment, teams are working together on
```

Comment:
software development -> machine learning 

(I even agree that it's software engineering, but it's bette to delineate them for now)